frontend/src/metabase/components/form/CustomForm/CustomFormFooter/CustomFormFooter.styled.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from "@emotion/styled";
+
+interface CustomFormFooterStyledProps {
+  shouldReverse?: boolean;
+}
+export const CustomFormFooterStyled = styled.div<CustomFormFooterStyledProps>`
+  display: flex;
+  flex-align: center;
```

Comment:
This is a typo, there's no such CSS property. This cause the UI button to go full width.
```suggestion
  align-items: center;
```
#### Before
![image](https://user-images.githubusercontent.com/1937582/177138766-f3f81b71-8539-4c7a-a443-b6ee1866247d.png)

#### After
![image](https://user-images.githubusercontent.com/1937582/177138535-6c5c9a70-6ad4-4c74-a224-91f85f5540af.png)